### PR TITLE
Replace deprecated storage account diagnostic settings policy in initiative

### PR DESCRIPTION
**Pull request by chrholt**
## Overview/Summary

Replace deprecated diagnostic setting policy for Storage Account with replacements in `Deploy-Diagnostics-LogAnalytics` initiative

## This PR fixes/adds/changes/removes

1. Removes deprecated diagnostic setting policy for Storage Account (Definition ID: 6f8f98a4-f108-47cb-8e98-91a0d85cd474)

This has been replaced with the following 5 separate policy definitions:

- Storage Accounts (DefinitionID: 59759c62-9a22-4cdf-ae64-074495983fef)
- Blob Services (DefinitionID: b4fe1a3b-0715-4c6c-a5ea-ffc33cf823cb)
- File Services (DefinitionID: 25a70cc8-2bd4-47f1-90b6-1478e4662c96)
- Queue Services (DefinitionID: 7bd000e3-37c7-4928-9f31-86c4b77c5c45)
- Table Services (DefinitionID: 2fb86bf3-d221-43d1-96d1-2434af34eaa0)

## Testing Evidence

Please provide any testing evidence to show that your Pull Request works/fixes as described and planned (include screenshots, if appropriate).

### Testing URLs

The below URLs can be updated where the placeholders are, look for `{YOUR GITHUB BRANCH NAME HERE - Remove Curly Brackets Also}` & `{YOUR GITHUB BRANCH NAME HERE - Remove Curly Brackets Also}`, to allow you to test your portal deployment experience.

> Please also replace the curly brackets on the placeholders `{}`

#### Azure Public

[Deploy To Azure](https://portal.azure.com/#blade/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fchrholt%2FEnterprise-Scale%2Freplace-storage-account-diagnostic-policies%2FeslzArm%2FeslzArm.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2Fchrholt%2FEnterprise-Scale%2Freplace-storage-account-diagnostic-policies%2FeslzArm%2Feslz-portal.json)

#### Azure US Gov (Fairfax)
[Deploy To Azure](https://portal.azure.us/#blade/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fchrholt%2FEnterprise-Scale%2Freplace-storage-account-diagnostic-policies%2FeslzArm%2FeslzArm.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2Fchrholt%2FEnterprise-Scale%2Freplace-storage-account-diagnostic-policies%2FeslzArm%2Ffairfaxeslz-portal.json)

## As part of this Pull Request I have

- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Enterprise-Scale/pulls)
- [x] Associated it with relevant [issues](https://github.com/Azure/Enterprise-Scale/issues), for tracking and closure.
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Enterprise-Scale/tree/main)
- [ ] Performed testing and provided evidence.
- [ ] Updated relevant and associated documentation.
- [x] Updated the ["What's New?"](https://github.com/Azure/Enterprise-Scale/wiki/Whats-new) wiki page (located: `/docs/wiki/whats-new.md`)

### CODE REVIEW

**Comment by chrholt** [comment]
#1034

File: docs/wiki/Whats-new.md [hunk 1]
```diff
@@ -64,7 +64,8 @@ Here's what's changed in Enterprise Scale/Azure Landing Zones:
 
 ### Policy
 
-- "**Deploy Diagnostic Settings for Log Analytics to Log Analytics workspace**" definition added and also added to `Deploy-Diagnostics-LogAnalytics` initiative 
+- Updated "**Deploy Diagnostic Settings to Azure Services**" initiative replacing deprecated policy for diagnostic settings on Storage Account
```
Comment (jtracey93) [inline docs/wiki/Whats-new.md:67]: @chrholt can you move this into the new December section of the Whats New that was added yesterday in #1143

**Review by jtracey93 — CHANGES_REQUESTED** [review]

**Review by jtracey93 — CHANGES_REQUESTED** [review]
Sorry @chrholt could you also increment the version number https://github.com/Azure/Enterprise-Scale/pull/1146/files#diff-50daedcf0a6f27ed5bd5e9b7670ffc48ae0433a7ea7ef351820d94229ee0f6f3R11) of the policy initiative to `2.0.0` as per https://github.com/Azure/azure-policy/tree/f4b95514ffb8d7a638c8a5d3ccb15a59031556bd/built-in-policies#versioning

**Comment by chrholt** [comment]
> Sorry @chrholt could you also increment the version number https://github.com/Azure/Enterprise-Scale/pull/1146/files#diff-50daedcf0a6f27ed5bd5e9b7670ffc48ae0433a7ea7ef351820d94229ee0f6f3R11) of the policy initiative to `2.0.0` as per https://github.com/Azure/azure-policy/tree/f4b95514ffb8d7a638c8a5d3ccb15a59031556bd/built-in-policies#versioning

Of course. My bad, should've thought of that.

**Review by jtracey93 — APPROVED** [review]
LGTM thanks @chrholt :shipit:

**Pull request merged**